lib/chunkifier.js

```diff
@@ -0,0 +1,37 @@
+const chunkifyArray = (messageLines, maxLength, delimiter) => {
+  delimiter = delimiter || '\n'
+
+  let message = ''
+  const resultLines = []
+  messageLines.forEach(line => {
+    let chunk = line + delimiter
+    if ((message + chunk).length > maxLength) {
+      resultLines.push(message.trimEnd())
+      message = chunk
```

Comment:
what does this line do?

Reply:
I should probably explain the algorithm "inline". I'll add that when I add docs and tests.

But basically we have `messageLines` which is an array of all the text split newline (`\n`) and we have the `maxLength` which is the size of each message `chunk`

So for each item in `messageLines` we test if adding a new `chunk` will put us _over_ the limit (`maxLength`). If it does we "reset" `message` with this new chunk (line 10).

If it doesn't put us over, we keep concatenating the chunks to `message`.

Hope that makes sense? 😂